.husky/post-merge

```diff
@@ -1,5 +1,11 @@
 #!/bin/sh
 
+if [ -z "$ENABLE_AUTO_SYNC_ON_PULL" ]
+then
```

Comment:
what is this for?

Reply:
I'm not a big fan of a post hook that does that much work.

Reply:
Given how rarely the apps change let's start with just the script for now :pray: